azurePipeline.yml

```diff
@@ -62,33 +75,37 @@ stages:
         Python3.7:
           pythonVersion: '3.7'
           artifactName: 'wheels.linux37'
+        Python3.8:
+          pythonVersion: '3.8'
+          artifactName: 'wheels.linux38'
     steps:
       - template: .azurePipeline/unittest_wheel_steps.yml
         parameters:
           artifactName: $(artifactName)
           pythonVersion: $(pythonVersion)
           operatingSystem: 'ubuntu-16.04'
-#  - job:
-#    displayName: MacOS
-#    pool:
-#      vmImage: 'macOS-10.13'
-#    strategy:
-#      matrix:
-#        Python3.6:
```

Comment:
So we don't support Python 3.6 anymore?

Reply:
We will, but it is currently (still) broken until the 220 PR is merged and released in cibuildwheel